ci/check-dist.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+VERSION=`git describe`
+
+make dist || exit 1
+tar zxvf sillymud-$VERSION.tar.gz || exit 1
+(cd sillymud-$VERSION; CFLAGS="-std=gnu99" ./configure && make) || exit 1
```

Comment:
Just as a matter of style, I tend to put `set -e` at the top of scripts instead of `|| exit 1` everywhere. Otherwise :+1: